scripts/watchWithPolling.mjs

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
why does this file exist?

Reply:
Check my previous comment. It watches in polling mode, because the normal watch doesn't work in docker.